docs/sources/enterprise/usage-insights.md

```diff
@@ -0,0 +1,50 @@
++++
+title = "Usage-insights"
+description = "Usage-insights"
+keywords = ["grafana", "usage-insights", "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/usage-insights/"]
+type = "docs"
+[menu.docs]
+name = "Usage-insights"
+parent = "enterprise"
+weight = 700
++++
+
+# Usage Insights
+
+Usage Insights allows you to have a better understanding on how your Grafana instance is used. The collected data are: dashboard views, errors and queries count and user dashboard views.
+
+> Only available in Grafana Enterprise v7.0+.
+
+## Presence Indicators
+
+Presence indicators are visible to all signed-in users on all dashboards. It shows the most recent users who interacted with the dashboard.
+
+To get more information, click on the `+X` icon that opens the [Dashboard Insights]({{< relref "#dashboard-insights" >}}) drawer. You can see the user's name by hovering the user's icon. Users icons come from [Gravatar](https://gravatar.com).
```

Comment:
Rewrote to clarify that `+X` is not always present. 

```suggestion
When more users are active on a dashboard than can fit in the presence indicator, click on the `+X` icon to open the [Dashboard Insights]({{< relref "#dashboard-insights" >}}) drawer and see more details about recent user activity. You can see the user's name by hovering the user's avatar. Users' avatars come from [Gravatar](https://gravatar.com) based on the user's email.
```

Reply:
With your improvements, the order of the sentences felt a bit weird so I rewrote the paragraph instead of commiting your changes. Feel free to add more comments if it's not clear enough!

Reply:
:+1: I agree, your rewrite improves on my changes!